main/NetworkImage.tsx

```diff
@@ -23,7 +22,7 @@ type Styles = {
 interface Props {
   style?: StyleProp<ViewStyle>;
   styles?: Styles;
-  source: ImageSourcePropType | undefined;
+  uri: string | undefined;
```

Comment:
I think changing `uri` to `url` is more understandable in this case.

The terms “URI” and “URL” are often used interchangeably, but they are not exactly the same.
Followed by [reference](https://danielmiessler.com/study/difference-between-uri-url).

Here we are only handling network images which are urls.

Reply:
@hyochan  I got it!